### DISCUSSION TITLE
Problem B: More Complex Metadata

### DISCUSSION THREAD

**Discussion by BradKML** [cinemagoer/cinemagoer#340] [Ideas]
Seeing:
- Only Creator https://github.com/alberanid/imdbpy/issues/275
- Cast & Crew https://github.com/alberanid/imdbpy/issues/328
- Awards https://github.com/alberanid/imdbpy/issues/255

Data Object Format for Awards:
- Year/Season (Integer)
- Organization (String)
- Award Name (String)
- Movie/TV (Movie/TV Object)

Cast & Crew Format:
- Movie/TV (Movie/TV Object)
- General Role (String or Role-like Object)
- Staff (Person-like Object)
- Sub-divider in two forms
   - Character (String or Object)
   - Specific Title (String)

**Comment by BradKML** [comment]
@alberanid This is something worth discussing since it is useful data for those who would like to research genre and cultural transmission.

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
